internal/issuer/clients/cert/requests.go

```diff
@@ -19,20 +21,28 @@ const (
 	contentTypeHeaderKey    = "Content-Type"
 	contentTypeTextPlainKey = "text/plain"
 	csrKey                  = "csr"
+	fieldName               = "field"
+	fileName                = "csr.pem"
```

Comment:
May need to change these constant names, or maybe more them to the API so that they are configurable from outside the code.

Reply:
change `fieldName` to `file`